skimage/_shared/_interpolation_test.pyx

```diff
@@ -0,0 +1,31 @@
+from interpolation cimport coord_map as _coord_map
```

Comment:
Ah, ok. I should have looked further below, you already created the unit test. So, forget the last line... :-)